src/main/java/org/jabref/gui/groups/GroupTree.fxml

```diff
@@ -39,4 +29,15 @@
             </columnResizePolicy>
         </TreeTableView>
     </center>
+    <bottom>
+        <HBox fx:id="groupBar" alignment="CENTER" spacing="8">
+            <Button fx:id="addNewGroup" onAction="#addNewGroup" styleClass="add-group-button"
+                    text="%Add group" maxWidth="1.7976931348623157E308" HBox.hgrow="ALWAYS">
```

Comment:
I think you can remove the maxWidth property

Reply:
You can also assign infinity in the fxml directly maxWidth="Infinity

Reply:
done thank you.